docs/exchange-operators/faq.mdx

```diff
@@ -188,13 +188,13 @@ You can look into staking this wallet, either by running your own block producti
 
 :::note
 
-Staking will incur a delay between 18 to 29 days before you will start receiving rewards.
+Staking incurs a delay between 18 to 29 days before you start receiving rewards.
```

Comment:
but w/o the hyphen, since we don't hyphenate after "ly" 
How do I know this? My IBM editor corrected my resume decades ago (oof)

> Part of the reasoning for not using a hyphen after “-ly” adverbs is that, appearing before a verb as it does, an “-ly” adverb is obviously attached to it, so no confusion is possible. As The Chicago Manual of Style says: “Compounds formed by an adverb ending in ly plus an adjective or participle (such as largely irrelevant or smartly dressed) are not hyphenated either before or after a noun, since ambiguity is virtually impossible. (The ly ending with adverbs signals to the reader that the next word will be another modifier, not a noun.)”

Reply:
oh cool